From 289e06eec7a6099d54b62cded5f14c1a9d5a3dd7 Mon Sep 17 00:00:00 2001
From: Daniele Di Proietto <ddiproietto@google.com>
Date: Fri, 3 Dec 2021 13:31:55 +0000
Subject: [PATCH] Fix sc-dev-plus-aosp builds

Ignore-AOSP-First: Fix merge conflict
Bug: 208964910
Change-Id: I7924d7b19ec30e42fd1d17ec913c4a75d51ea6bc
(cherry picked from commit 823f787a31bf7826add4104c31c76e39981a0369)
Merged-In:I7924d7b19ec30e42fd1d17ec913c4a75d51ea6bc
---
 aidl_unittest.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aidl_unittest.cpp b/aidl_unittest.cpp
index 4621dd80..f825cfd2 100644
--- a/aidl_unittest.cpp
+++ b/aidl_unittest.cpp
@@ -32,6 +32,7 @@
 #include "aidl_language.h"
 #include "aidl_to_cpp.h"
 #include "aidl_to_java.h"
+#include "aidl_to_ndk.h"
 #include "comments.h"
 #include "logging.h"
 #include "options.h"
-- 
2.17.1

